website/content/docs/commands/secrets/tune.mdx

```diff
@@ -57,8 +57,8 @@ flags](/docs/commands) included on all commands.
 
 - `-max-lease-ttl` `(duration: "")` - The maximum lease TTL for this secrets
   engine. If unspecified, this defaults to the Vault server's globally
-  configured maximum lease TTL, or a previously configured value for the secrets
-  engine.
+  configured [maximum lease TTL](/docs/configuration#max_lease_ttl), or a previously configured value for the secrets
+  engine. This value is allowed to override the server's global max TTL; it can be longer or shorter.
```

Comment:
Judging from https://www.vaultproject.io/docs/commands/auth/tune#max-lease-ttl, I think we should make the same edit there?

Reply:
yep

Reply:
sorry, closed this w/o that change. that change was just made https://github.com/hashicorp/vault/pull/12391/files#diff-3dd98f20e81f7ae91d858fc307fae5185a1c8a8ddf9e8a4fa91c708797dd1d25